packages/core/utils/workspace.ts

```diff
@@ -71,6 +72,11 @@ export async function populateWorkspaceDetails(workspace: WorkspaceWithoutExplic
         workspace.typescript.installed = "tslib" in packageJson.devDependencies;
         workspace.prettier.installed = "prettier" in packageJson.devDependencies;
         workspace.kit.installed = "@sveltejs/kit" in packageJson.devDependencies;
+        workspace.dependencies = { ...packageJson.devDependencies, ...packageJson.dependencies };
```

Comment:
I'm honestly not sure if we should mix deps and dev deps here. But tbh, I cannot imagine any reasoning here why this should not be the case. So feel free to resolve this.

Reply:
Yea I couldn't think of any scenario where being a dep or devDep would make a difference, so I merged them together for easier usage